calyx-opt/src/passes/cell_share.rs

```diff
@@ -202,110 +233,6 @@ impl CellShare {
         }
     }
 
-    // given a ctx, gets the
-    // 1) file to write the sharing frequencies. for example, if "-x cell-share:print-share-freqs=a.json",
-    // we would return Some(a.json).
-    // 2) gets the bounds. For example, if "-x cell-share:bounds=2,3,4" is passed
-    // we would return [Some(2),Some(3),Some(4)].
-    // 3) whether to print the par timing map. For exampe, if "-x cell-share:print_par_timing"
-    ///4) whether to run sharing with Calyx 2020 settings: no component sharing,
-    /// only share registers/combinational components (e.g., "-x cell-share:calyx_2020")
-    /// 5) whether to share across static par threads (e.g., "-x cell-share:share_static_par")
```

Comment:
@calebmkim it seems that this option is not actually used (or even parsed) in this function. Is that expected?

Reply:
Yeah you're right. It just always shares across static pars now. (I think this was from an earlier version of the pass and I forgot to change the documentation).

Reply:
Cool, any point adding this option back?